CUJExecutor/yesProfileCuj

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
There's quite a bit of repetition.  Should compose the bash scripts from smaller scripts.  Or even better, migrate to python/java/go.